src/haddock/core/supported_molecules.py

```diff
@@ -1,10 +1,46 @@
 """
 HADDOCK 3 supported residues.
 
-https://bianca.science.uu.nl/haddock2.4/library
-
-
-Write decent documentation .-)
```

Comment:
😆

Reply:
Made the general `.top` file reader as you suggested :+1:. Ended up using just a small regex for the atom lines, else did with python.

Reply:
🥲